doc/dependencies.md

```diff
@@ -0,0 +1,38 @@
+Dependencies
+============
+
+These are the dependencies currently used by Bitcoin Core. You can find instructions for installing them in the `build-*.md` file for your platform.
+
+| Dependency | Version used | Minimum Required | Latest available | CVEs? | Shared | [Bundled Qt Library](https://doc.qt.io/qt-5/configure-options.html) |
```

Comment:
Let's not try to track latest available here...

For CVEs, rather than yes/no, it'd be more useful to link where to check for exploits.